pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -778,6 +778,48 @@ def test_dt64arr_iadd_timedeltalike_scalar(
         rng += two_hours
         tm.assert_equal(rng, expected)
 
+    @pytest.mark.filterwarnings(
+        "ignore:Adding/subtracting object-dtype array to DatetimeArray"
+    )
+    @pytest.mark.parametrize(
+        "dt64arr, offset_arr, expected",
```

Comment:
if im reading this right, this is a length-1 list?  if so, just put it in the test